### ISSUE TITLE
PC 2 PC is not working

### ISSUE THREAD

**Issue by unruledboy** [ianramzy/decentralized-video-chat#25]
Hi,

Very neat solution, love it!

It works for my phone (Safari) vs PC (new Edge), but it does not work with PC 2 PC, both uses chrome.

I can tell both PCs are behind home routers, but it should not be a problem, right？

**Comment by Chaphasilor** [comment]
No, this should indeed not be an issue. Could you maybe provide some additional information? browser version, input devices, etc.

Also, could you try if this works in other configurations, like your phone and you friend's PC, or vice versa, and see if that works?
